### DISCUSSION TITLE
Does lazy.nvim integrate any plugins for commenting code?

### DISCUSSION THREAD

**Discussion by lainegates** [LazyVim/LazyVim#2415] [Q&A] [answered]
I searched all menus under '<leader>' (i.e. <space>), I didn't find the command to comment code.

I also searched "https://www.lazyvim.org/", I didn't find the command either.

I installed [omment.nvim](https://github.com/numToStr/Comment.nvim) , but the default shortcut keys don't seem to align with lazy.nvim's logic. Due to that commenting code is a very common requirement, and I want to confirm whether I have not found the correct settings in lazy.nvim?

**Comment by dpetka2001** [comment, marked as answer]
LazyVim by default has [mini.comment](https://www.lazyvim.org/plugins/coding#minicomment) for commenting code. The default prefix for commenting is `gc`. If you press that you can see different objects for commenting. For example, `gcc` is used for commenting line.
